Makefile

```diff
@@ -169,47 +169,24 @@ deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in
 		&& $(KUSTOMIZE) edit set namespace ${OPERATOR_NS}
 	$(KUSTOMIZE) build config/overlays/make-deploy | kubectl apply -f -
```

Comment:
Pretty sure you will need `$(KUSTOMIZE) build config/overlays/odh | kubectl apply -f -`  here as `deploy` isn't deploying anything from `argo`

Reply:
Ignore this comment, and actually the modification I suggested for `make-deploy/kustomization.yaml` will be a better approach